index.html

```diff
@@ -1507,182 +1365,59 @@ <h1>FOSSASIA Summit Young Developers Day</h1>
-
                         <div class="col-md-4 col-sm-6 sponsor-column">
                             <div class="sponsor text-center">
                                 <a href="https://lli.sg" target="_self">
                                     <img alt="Lifelong Learning Institute" src="img/llisg-logog.png">
                                 </a>
                             </div>
                         </div>
-
                         <div class="col-md-4 col-sm-6 sponsor-column">
                             <div class="sponsor text-center">
                                 <a href="http://mysql.org" target="_self">
                                     <img alt="MySQL" src="img/MysqlOracle.png">
                                 </a>
                             </div>
                         </div>
-
                         <div class="col-md-4 col-sm-6 sponsor-column">
                             <div class="sponsor text-center">
                                 <a href="http://mbm.asia" target="_self">
                                     <img alt="MBM" src="img/mbm_logo.png">
                                 </a>
                             </div>
                         </div>
-
                         <div class="col-sm-12 text-center">
                             <span>Interested in becoming a sponsor or exhibitor? <a href="mailto:contact@fossasia.org" target="default">Get in touch</a> or book an <a href="./exhibitors/index.html" target="default">exhibition package here</a>.</span>
                         </div>
                     </div>
-
-                    <!--            <div class="col-md-4 col-sm-6 sponsor-column">
-                                <div class="sponsor text-center">
-                                    <a href="https://www.google.com/cloud/" target="default">
-                                        <img alt="Sponsor" src="img/google.png">
-                                    </a>
-                                </div>
-                            </div>
-                                <div class="col-md-4 col-sm-6 sponsor-column">
-                                <div class="sponsor text-center">
-                                    <a href="https://daimler.com/" target="default">
-                                        <img alt="Sponsor" src="img/Daimler_logo_gray.png">
-                                    </a>
-                                </div>
-                            </div>
-                                <div class="col-md-4 col-sm-6 sponsor-column">
-                                <div class="sponsor text-center">
-                                    <a href="http://microsoft.com/" target="default">
-                                        <img alt="Sponsor" src="img/microsoft.png">
-                                    </a>
-                                </div>
-                            </div>
-                            <div class="col-md-4 col-sm-6 sponsor-column">
```

Comment:
Why was this huge block deleted?

Reply:
Because it's really of no use and it can be regenerated according to the requirement again. It was a section in previous summit but not included in this. Just a chunk of space holder.